src/Proxies/WP.php

```diff
@@ -32,7 +32,7 @@ class WP {
 	 */
 	public function __construct() {
 		global $wp;
-		$this->wp =& $wp;
+		$this->wp = & $wp;
```

Comment:
Any particular reason this needs to be changed? It's assigning by reference so the preference is to write it as `=&`.
Was it maybe a global search and replace when adding a space before & for this line which changed it?
`esc_js( join( ' & ', $this->product_helper->get_categories( $product ) ) )`

Reply:
This is a typo which caused by format. Fixed.